src/test/suite/lib/ggshield-api.test.ts

```diff
@@ -78,19 +78,22 @@ suite("scanFile", () => {
       statusBar.StatusBarStatus.ignoredFile
     );
   });
-
-  test("displays an error message if the scan command fails", async () => {
-    runGGShieldCommandMock.returnWith({
-      status: 1,
-      stdout: "",
-      stderr: "Error",
+  
+  const errorCodes = [128, 3];
+  errorCodes.forEach((code) => {
+    test(`displays an error message if the scan command fails with error code ${code}`, async () => {
+      runGGShieldCommandMock.returnWith({
+        status: code,
+        stdout: "",
+        stderr: "Error",
+      });
+
+      await scanFile("test.py", Uri.file("test.py"), {} as GGShieldConfiguration);
+
+      // The error message is displayed
+      assert.strictEqual(errorMessageMock.callCount, 1);
+      assert.strictEqual(errorMessageMock.lastCall.args[0], "ggshield: Error\n");
```

Comment:
The test now fails because there is no `\n` at the end of the test anymore.

Reply:
And... We really need a CI :sweat: